examples/kubernetes-kafka/kafka-sw-app.yaml

```diff
@@ -103,6 +116,7 @@ spec:
       labels:
         app: empire-outpost
         outpostid: "8888"
+        zgroup: kafkaTestApp
```

Comment:
If you are going to change this YAML, please confirm that this does not break the existing GSG by manually testing the steps in the GSG with this YAML. When you do that, please paste the output from your terminal on this PR to show you have performed this manual testing.